app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def format_currency(amount)
+    number_to_currency(amount).sub(/\.00$/, '')
```

Comment:
`number_to_currency` takes a `precision` option - http://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_currency

You should be able to drop that in the locale and do away with the customer helper.